solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1521,6 +1524,35 @@ impl pallet_registrar::RegistrarHooks for StarlightRegistrarHooks {
     }
 }
 
+pub struct BabeSlotBeacon;
+
+impl BlockNumberProvider for BabeSlotBeacon {
+    type BlockNumber = u32;
+
+    fn current_block_number() -> Self::BlockNumber {
+        // TODO: nimbus_primitives::SlotBeacon requires u32, but this is a u64 in pallet_babe, and
+        // also it gets converted to u64 in pallet_author_noting, so let's do something to remove
+        // this intermediate u32 conversion, such as using a different trait
+        u64::from(pallet_babe::CurrentSlot::<Runtime>::get()) as u32
+    }
+}
+
+impl pallet_author_noting::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type ContainerChains = ContainerRegistrar;
+    type SlotBeacon = BabeSlotBeacon;
+    type ContainerChainAuthor = TanssiCollatorAssignment;
+    // We benchmark each hook individually, so for runtime-benchmarks this should be empty
+    #[cfg(feature = "runtime-benchmarks")]
```

Comment:
Since the associated type definition is same whether `runtime-benchmarks` feature is enabled or not, can we combine them into one?

Reply:
Better not to, because when we add something to the hook we only want to add it under `#[cfg(not(feature = "runtime-benchmarks"))]`.